bot.py

```diff
@@ -1,107 +1,120 @@
 import asyncio
 import logging
 import sys
-import traceback
+import typing
+from typing import Annotated, Optional, Literal
 
 import discord
+from discord.ext import commands
 from dynaconf import ValidationError
 
+import constants
 import lib
-from messages import SubmitMessage, GetBestTimesMessage
-from database import Database
 from config import settings
-import constants
+from database import Database
+from error_handler import ErrorHandlerCog
 
 logger = logging.getLogger(__name__)
 
 
-class MyBot(discord.Client):
+class MyBot(commands.Bot):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.queue = asyncio.Queue()
         self.db = Database().get()
 
+    async def setup_hook(self):
+        await asyncio.gather(
+            bot.add_cog(Commands(bot)),
+            bot.add_cog(ErrorHandlerCog(bot))
+        )
+
     async def on_ready(self):
         print("Logged in as", self.user)
         while True:
             try:
                 submit_msg = await self.queue.get()
                 logger.info("Going to process submission from queue: %s", submit_msg)
-                await lib.benchmark(submit_msg)
+                await lib.benchmark(*submit_msg)
                 self.queue.task_done()
             except Exception:
                 logger.exception("Error while processing submission.")
 
-    async def handle_best(self, msg):
+
+# if i don't use a cog, the functions would need to be in __name__ == __main__
+class Commands(commands.Cog):
+    def __init__(self, sbot: MyBot):
+        self.bot = sbot
+
+    @commands.command()
+    @commands.is_owner()
+    async def sync(self, ctx: commands.Context):
+        # sync slash commands
+        # this is a separate command because rate limits
+        await self.bot.tree.sync()
+        await ctx.reply("synced!")
+
+    # intentionally did not use typing.Optional because dpy treats it differently and i dont want that behavior
+    @commands.hybrid_command(aliases=["aoc", "lb"])
```

Comment:
personally i think this should just be `lb`, aoc is like a prefix, best is vague too

Reply:
I dont think having extra aliases hurts anything